### PR TITLE
adds weekly Docker image updater

### PR DESCRIPTION
**Pull request by gedigi**
### Description
Adds a script and workflow that checks the latest digest of the Aptos container and updates the Dockerfile as needed. The workflow runs weekly.

### Test Plan
The workflow runs when a pull request includes its source or the related script, and will produce an error if it doesn't pass.

### CODE REVIEW

File: .github/workflows/docker-update-images.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Continuously build Rosetta
+name: Docker Update Images
+
+on:
```
Comment (perryjrandall) [inline .github/workflows/docker-update-images.yaml:4]: Add a "pull request" that matches this file and the python file path

That way your PR will actually run and test this workflow

File: .github/workflows/docker-update-images.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  schedule:
+    - cron: "0 9 * * 1" # once a week
+
+jobs:
+  update:
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Check and pin updates (if any)
```
Comment (perryjrandall) [inline .github/workflows/docker-update-images.yaml:13]: You might need to add setup python step? Although without dependencies it might just work tm

**Review by perryjrandall — APPROVED** [review]
Generally LGTM, please add a test plan

File: .github/workflows/docker-update-images.yaml (hunk 1 above)
Comment (perryjrandall) [reply .github/workflows/docker-update-images.yaml:4]: Also you'll wanna add a workflow_dispatch so that it can be triggered manually as well

**Review by sionescu — APPROVED** [review]

**Pull request merged**